casts/free-podcasts-screencasts-pt_PT.md

```diff
@@ -8,11 +8,13 @@
 ### Desenvolvimento Web
 
 * [10webPodcast sobre web e desenvolvimento em português](https://10web.pt/acerca) - Ricardo Correia, Vitor Silva, Ana Sampaio (podcast)
+* [Desenvolvimento de sites, muito além da programação - O2 Podcast #1](https://www.youtube.com/watch?v=EJu14PUvRMs) - Oxigenweb - Agência Digital
 
 
 ### Laravel
 
 * [Laravel Portugal Live](https://laravelportugal.simplecast.fm) (screencast)
+* [O QUE TODO DESENVOLVEDOR DEVERIA SABER SOBRE O LARAVEL | PODCAST PAPO WEB #055](https://www.youtube.com/watch?v=F5nXYbqAOFQ) - Robson V. Leite
```

Comment:
no ALLCAPS titles, escape `|`.
For youtube, we prefer "playlist" links and the name of the producer/creator. docs/CONTRIBUTING.md#platforms-and-access-notes